enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts

```diff
@@ -38,3 +26,19 @@ export const EMBEDDING_FAILED_MESSAGE = `
   Failed to enable embedding features.
   ${DELETE_CONTAINER_MESSAGE}
 `;
+
+// eslint-disable-next-line no-unconditional-metabase-links-render -- link for the CLI message
+export const METABASE_INSTANCE_SETUP_COMPLETE_MESSAGE = `
```

Comment:
Consolidate the instance setup success and the anonymous tracking information in the same message, as this is shown as the final step